<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-K853HKP" failures="1" tests="6" name="loginapp" time="0.192" errors="0" timestamp="2023-09-25T17:53:33 IST">
  <testcase classname="testnglearning.test3hello" name="applogin" time="0.05">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testnglearning.test3hello.applogin(test3hello.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- applogin -->
  <testcase classname="testnglearning.test3hello" name="applogin1" time="0.004"/>
  <testcase classname="testnglearning.test3hello" name="applogin2" time="0.002"/>
  <testcase classname="testnglearning.test3hello" name="applogin3" time="0.01"/>
  <testcase classname="testnglearning.test3hello" name="applogin3" time="0.003"/>
  <testcase classname="testnglearning.test3hello" name="applogin3" time="0.002"/>
</testsuite> <!-- loginapp -->
